resources/lang/fr/messages.php

```diff
@@ -142,10 +142,8 @@
     'outpost_error_422' => 'Erreur de communication avec statamic.com.',
     'outpost_error_429' => 'Trop de demandes à statamic.com.',
     'outpost_issue_try_later' => 'Un problème est survenu lors de la communication avec statamic.com. Veuillez réessayer plus tard.',
-    'password_protect_enter_password' => 'Entrez le mot de passe pour déverrouiller',
     'password_protect_incorrect_password' => 'Mot de passe erroné',
     'password_protect_token_invalid' => 'Jeton non valide ou ayant expiré',
-    'password_protect_token_missing' => 'Le jeton sécurisé fait défaut. Vous devez arriver sur cet écran depuis l’URL protégée originelle.',
```

Comment:
These shouldn't be removed. Did the translator tool do that?

Reply:
Yes it did, I just checked this. I'll make another commit to get those back then.